docs/UserGuide.adoc

```diff
@@ -149,6 +149,19 @@ Example:
 
 * `register n/matt p/83938249 e/matt@damon.com`
 
+==== Unregistering a borrower: `unregister`
+
+Unregisters a borrower from the library records.
+Format:  `unregister id/BORROWER_ID`
+
+* Deletes the book with the specified borrower ID
```

Comment:
I think we need to change `book` to `borrower`

Reply:
okay! careless whoops